crates/shadowsocks-service/src/local/dns/server.rs

```diff
@@ -315,7 +315,8 @@ fn check_name_in_proxy_list(acl: &AccessControl, name: &Name) -> Option<bool> {
         // remove the last dot from FQDN
         let mut name = name.to_ascii();
         name.pop();
```

Comment:
I think this line (and comment above) might be deleted now, since last dot is deleted in `check_host_matched`.

Reply:
You are right.